metrics.py

```diff
@@ -0,0 +1,102 @@
+"""Collection of metrics for evaluating the performance of probabilistic models."""
+
+from arviz_base import convert_to_datatree, dataset_to_dataframe, extract, rcParams
+from scipy.stats import wasserstein_distance, wasserstein_distance_nd
+
+
+def wasserstein(
+    data_1,
+    data_2,
+    group="posterior",
+    var_names=None,
+    sample_dims=None,
+    joint=True,
+    num_samples=500,
+    random_seed=212480,
+):
+    """
+    Compute the Wasserstein-1 distance.
+
+    Parameters
+    ----------
+    data : Dataset, DataTree, or InferenceData
```

Comment:
```suggestion
    data_1, data_2 : Dataset, DataTree, or InferenceData
```

numpydoc allows parameters that have the same type and similar description to be documented in the same line. Ref: https://numpydoc.readthedocs.io/en/latest/format.html#parameters

> When two or more input parameters have exactly the same type, shape and description, they can be combined:
>
>     x1, x2 : array_like
>         Input arrays, description of `x1`, `x2`.